dietpi/dietpi-software

```diff
@@ -721,6 +722,40 @@ INDEX_WEBSERVER_TARGET=${INDEX_WEBSERVER_TARGET:=-2}
 			fi
 
 		fi
+		
+		#OPENBAZAAR
+		if (( $OPENBAZAAR == 1 )); then
+		
+			INSTALL_DESCRIPTION='OpenBazaar'
+			Banner_Installing
+		    
+			#apt-get -y update && apt-get -y upgrade
+			apt-get -y install git
+			apt-get -y install build-essential libssl-dev libffi-dev python-dev openssl python-pip libzmq3-dev
```

Comment:
```build-essential``` and ```git``` should trigger the DietPi installations of them in this script 

Add entry for ```OPENBAZAAR``` to trigger these.
```
		#DietPi software that requires GITCLIENT
		if (( $EMONCMS == 1 )); then
			GITCLIENT=1
		elif (( $AMPACHE == 1 )); then
			GITCLIENT=1
		fi

		#DietPi software that requires Build-Essential
		if (( $HAPROXY == 1 )); then
			BUILDESSENTIAL=1
		elif (( $WIRINGPI == 1 )); then
			BUILDESSENTIAL=1
		fi

```